components/style/properties/properties.mako.rs

```diff
@@ -820,11 +821,14 @@ pub enum DeclaredValue<'a, T: 'a> {
 /// extra discriminant word) and synthesize dependent DeclaredValues for
 /// PropertyDeclaration instances as needed.
 #[derive(Clone, PartialEq, Eq, Debug)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 pub enum DeclaredValueOwned<T> {
     /// A known specified value from the stylesheet.
     Value(T),
     /// An unparsed value that contains `var()` functions.
-    WithVariables(Arc<UnparsedValue>),
+    WithVariables(
```

Comment:
nit: I think this would look nicer changing it to:

```rust
WithVariables {
    #[cfg_attr(feature = "servo", ...)]
    value: Arc<UnparsedValue>,
}
```